src/envo/envo-edit.owl

```diff
@@ -37272,6 +37302,57 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000134> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000134> "thermal reactor")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000134> "thermal nuclear reactor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000134> <http://purl.obolibrary.org/obo/ENVO_02000137>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000135> (fast neutron reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Fast-neutron_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000135> "A nuclear reactor in which the fission chain reaction is sustained by fast neutrons.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000135> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000135> "fast reactor")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000135> "fast neutron reactor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000135> <http://purl.obolibrary.org/obo/ENVO_02000137>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000136> (research nuclear reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Research_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000136> "A nuclear reactor which serves primarily as a neutron source.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000136> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000136> "non-power reactor")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000136> "nuclear test reactor")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000136> "research nuclear reactor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000136> <http://purl.obolibrary.org/obo/ENVO_02000133>)
+DisjointClasses(<http://purl.obolibrary.org/obo/ENVO_02000136> <http://purl.obolibrary.org/obo/ENVO_02000137>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000137> (nuclear power reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Nuclear_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000137> "A nuclear reactor which is used as a source of power.")
```

Comment:
Seems circular, but we do have the power plant branch - could that be used / cross-axiomatised here? Like a power plant that has part some nuclear reactor?

Reply:
it sounds circular but it's not, as the differentia is in the intended use/function, to produce power (as opposed to say producing radioisotopes for imaging). We do have a logical axiom connecting the plane to the reactor.

I do think it's worth doing an overall review of design patterns for the existing terms for power plant - do you mind if we do this as a separate ticket/PR?

Reply:
>it sounds circular but it's not, as the differentia is in the intended use/function, to produce power (as opposed to say producing radioisotopes for imaging). We do have a logical axiom connecting the plane to the reactor.

Okay, clarified. 


>I do think it's worth doing an overall review of design patterns for the existing terms for power plant - do you mind if we do this as a separate ticket/PR?

Yes, that should be separate to stay focused.